xml/decoders/piko/Dateiliste_piko_4.1.txt

```diff
@@ -0,0 +1,38 @@
+cv107.108_v4.1.xml
+cv109-123_v4.1.xml
+cv124.129_v4.1.xml
+cv12_v4.1.xml
+cv130.134_v4.1.xml
+cv15.16.xml
+cv150-157_v4.1.xml
+cv2-6_v4.1.xml
+cv257-512_v4.1.xml
+cv28.xml
+cv30error.xml
+cv321-336_v4.1.xml
+cv33-46_v4.1.xml
+cv337-352_v4.1.xml
+cv353-368_v4.1.xml
+cv369-384_v4.1.xml
+cv385-400_v4.1.xml
+cv47-49_v4.1.xml
+cv50.51_v4.1.xml
+cv53-63_v4.1.xml
+cv59factReset_v4.1.xml
+cv64.65_v4.1.xml
+cv900-1024_PIKO404ß1-18_v4.1.xml
+cv900-1024_PIKO46440-18_v4.1.xml
+cv900-1024_PIKO46442-16_v4.1.xml
+cv900-1024_PIKO46443-22_v4.1.xml
+cv900-1024_v4.1.xml
+enumFunctionChoiceF28Off255.xml
+enumPwm.xml
+PanePikoExtendedFmap_v4.1.xml
+PanePiko1.xml
+PanePiko2_v4.1.xml
+PanePikoAccelDecel.xml
+PanePikoFunctionMap_v4.1.xml
+PanePikoMotor.xml
+PanePikoRailcom.xml
+PanePikoSound_v4.1.xml
+

```

Comment:
@bobjacobsen 
Static analysis complains with:
```
Error:  Failures: 
Error:    FileLineEndingsCheck.lineEndings:122 File ./xml/decoders/piko/Dateiliste_piko_4.1.txt has incorrect line endings. ==> expected: <false> but was: <true>
Error:    FileLineEndingsCheck.lineEndings:122 File ./xml/decoders/piko/Dateiliste_piko_XP5.1.txt has incorrect line endings. ==> expected: <false> but was: <true>
Error:    FileLineEndingsCheck.lineEndings:122 File ./xml/decoders/Piko_SmartDecoder_4.1_Sound.xml has incorrect line endings. ==> expected: <false> but was: <true>
Error:    FileLineEndingsCheck.lineEndings:122 File ./xml/decoders/piko/cv897-1024_v5.1.xml has incorrect line endings. ==> expected: <false> but was: <true>
Error:    FileLineEndingsCheck.lineEndings:122 File ./xml/decoders/Piko_SmartDecoder_XP5.1_Sound.xml has incorrect line endings. ==> expected: <false> but was: <true>
```
But it seems to me that these files are fixed now. But Static analysis still fails. Can you see what's wrong?

Or do the files have CR+LF or CR instead of LF as line endings?

Reply:
I edit these files with the Notepad editor in Windows. I don't understand what happened. After all, the other files are all fine. Well, most of them are copies of existing files.

Thanks for your support and help